src/contracts/account/ReplayProtection.sol

```diff
@@ -38,7 +38,6 @@ contract ReplayProtection {
      *
      * Why is there no signing authority? An attacker can supply an address that returns a fixed signer
      * so we need to restrict it to a "pre-approved" list of authorities (DAO).
-
      * @param _callData Function name and data to be called
      * @param _replayProtectionAuthority What replay protection will we check?
      * @param _replayProtection Encoded replay protection
```

Comment:
saves on the .call() (e.g. 1k gas per transaction).

Reply:
e.g. if we assume either will be really popular, might as well have it native. i'm hoping the replay protection authority will be useful for other things (e.g. it could be a precondition check, or to verify a zkp).

Reply:
It wouldnt be a call though - since you're inheriting.

Reply:
i think given that the new ones are well-tested now, id just leave it the way it is tbh